app/routes/organization/$slug/index.tsx

```diff
@@ -365,26 +278,32 @@ export default function Index() {
           </div>
           {loaderData.userIsPrivileged && (
             <div className="absolute bottom-6 right-6">
-              <Form
-                method="post"
-                encType="multipart/form-data"
-                className="flex items-center"
-                reloadDocument
+              <label
+                htmlFor="modal-background-upload"
+                className="btn btn-primary modal-button"
               >
-                <InputImage
-                  id="background"
-                  name="background"
-                  maxSize={5 * 1024 * 1024} // 5 MB
-                  minWidth={1488} // 1488 px
-                  minHeight={480} // 480 px
-                  maxWidth={1920} // 1920 px
-                  maxHeight={1080} // 1080 px
-                  classes="opacity-0 w-0 h-0"
-                  containerRef={backgroundContainer}
-                  containerClassName="w-full h-full"
-                  imageClassName="object-cover w-full h-full"
-                />
-              </Form>
+                Bild ändern
+              </label>
+
+              <Modal id="modal-background-upload">
+                <ImageCropper
+                  headline="Hintergrundbild"
+                  subject="organisation"
+                  id="modal-background-upload"
+                  uploadKey="background"
+                  image={background}
```

Comment:
Reflect what is meant with aspect, minWidth, minHeight, targetWidth, targetHeight

f.e. 
- is minWidth the image width or the crop width
- same question with targetWidth, aspect, ...